packages/js/data/src/user/types.ts

```diff
@@ -40,12 +40,7 @@ export type WoocommerceMeta = {
 	[ key in keyof UserPreferences ]: string;
 };
 
-export type WCUser<
-	T extends keyof schema.Schema.BaseUser< 'view' > = schema.Schema.ViewKeys.User
-> = Pick<
-	schema.Schema.BaseUser< 'view' >,
-	schema.Schema.ViewKeys.User | T
-> & {
+export type WCUser< T extends Context = 'edit' > = User< T > & {
```

Comment:
The default `User` context is `edit` in `@wordpress/core-data` store. Using the same context for WCUser.